app/structuring_the_knight_personality.py

```diff
@@ -0,0 +1,20 @@
+from typing import List, Dict, Optional
```

Comment:
Rename the file, it is too wordy to store one `Knight` class.

Reply:
👍
![just_be_creative](https://github.com/mate-academy/py-knights-fighting/assets/108234997/64281382-a10d-4881-a04c-92835b664b25)